modules/sdk/src/main/scala/org/tessellation/sdk/domain/cluster/programs/Joining.scala

```diff
@@ -229,11 +229,11 @@ sealed abstract class Joining[F[_]: Async: GenUUID: SecurityProvider: KryoSerial
       _ <- nodeStorage.tryModifyState(NodeState.SessionStarted, stateAfterJoining).handleError(_ => ())
     } yield peer
 
-  private def validateWhitelisting(peer: PeerToJoin): F[Unit] =
-    whitelisting match {
+  private def validateSeedlist(peer: PeerToJoin): F[Unit] =
+    seedlist match {
       case None => Applicative[F].unit
       case Some(entries) =>
-        if (entries.contains(peer.id)) Applicative[F].unit else PeerNotWhitelisted(peer.id).raiseError[F, Unit]
+        if (entries.contains(peer.id)) Applicative[F].unit else PeerNotInSidelist(peer.id).raiseError[F, Unit]
```

Comment:
PeerNotInSeedList

Reply:
`Seedlist` is used like `Whitelist`

Reply:
That's okay, but there is a typo as well.